### PR TITLE
straw-man local dev with profile

### PR DESCRIPTION
**Pull request by ftdholbrook**
More of an RFC than a PR I want to get the direction I'm going in front of everyone

### CODE REVIEW

File: LOCAL-DEV.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+#Local Development Setup
```
Comment (malaffoon) [inline LOCAL-DEV.md]: I think we should introduce a `docs` folder. The main CONTRIBUTING.md will still be in root but then we can gut that and make it mostly a bunch of links to markdowns in `docs`.

Comment (millerts) [reply LOCAL-DEV.md]: Agreed.

File: LOCAL-DEV.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+###Environment Variables
+
+*TODO: should encourage the use of the current `${HOME}/smarter_app` location 
+perhaps with a `spring_config` sub-directory*
+```
+export SPRING_CONFIG_NAME=application,ap-iat-security
+export SPRING_CONFIG_LOCATION=file:///Users/<Your User Directory Name>/.spring_config/
+```
+
+###Configuration File
```
Comment (millerts) [inline LOCAL-DEV.md]: Note, Github requires a space after the last octothorpe for it to be treated as a heading.

**Comment by ftdholbrook** [comment]
@malaffoon, @millerts , @bdrainer I backed out the bulk of the changes to the application.yml files for now, so I believe these changes to be non breaking, and will only come into play with a local profile active.  My inclination is to go ahead and merge what I have now with 3 projects (item ingest, item search, and item bank sync) and continue to add projects as separate PRs or directly to the dev branch.  I have tested those 3 projects starting, but they need more in depth testing by someone who can actually run then and know how they work.

File: item-bank-sync-service/src/main/resources/application.yml [hunk 3]
```diff
@@ -18,7 +18,7 @@ management:
   context-path: "/manage"
 
 jwt:
-  secret: "\${TIMS_JWT_SECRET}"
```
Comment (ftdholbrook) [inline item-bank-sync-service/src/main/resources/application.yml]: I have never been able to get spring boot to accept this form of escape,  removing the quotes seems to work.

File: imrt-item-search-service/src/main/resources/application.yml [hunk 4]
```diff
@@ -23,6 +23,7 @@ management:
   security:
     enabled: "false"
 
+# TODO: seems like a bad thing for prod, should probably go in the local profile
```
Comment (malaffoon) [inline imrt-item-search-service/src/main/resources/application.yml]: 💯

File: item-bank-sync-service/src/main/resources/application.yml (hunk 3 above)
Comment (millerts) [reply item-bank-sync-service/src/main/resources/application.yml]: Yeah, I saw this as well.  I don't think the double quotes works and this is probably a bug.

**Pull request merged**